src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/ParametersTest.cs

```diff
@@ -902,13 +904,12 @@ public static void ClosedConnection_SqlParameterValueTest()
                             Assert.Fail($"Unexpected exception occurred: {e.Message}");
                         }
                     }
-                });
-                t.Start();
-                threads.Add(t);
+                }, TaskCreationOptions.LongRunning);
+                tasks.Add(t);
             }
-            for (int i = 0; i < threads.Count; i++)
+            for (int i = 0; i < tasks.Count; i++)
```

Comment:
This is basically the same as saying Task.WaitAll, right?

Reply:
Yes - agreed!